common/network-yarn/pom.xml

```diff
@@ -35,7 +35,7 @@
     <!-- Make sure all Hadoop dependencies are provided to avoid repackaging. -->
     <hadoop.deps.scope>provided</hadoop.deps.scope>
     <shuffle.jar>${project.build.directory}/scala-${scala.binary.version}/spark-${project.version}-yarn-shuffle.jar</shuffle.jar>
-    <shade>org/spark_project/</shade>
+    <shade>org/spark/project/</shade>
```

Comment:
I think this was on purpose, to avoid packaging into the org.spark namespace

Reply:
Probably doesn't matter much anyway, but to be more consistent with the original intent, `org/sparkproject` would be better.

Reply:
I recall we tried `spark-project` at some point and it was valid in bytecode but not in Java. `spark_project` ought to be a valid part of a package. Weak. OK I see the argument for `org.sparkproject`

Reply:
`spark_project` is valid, but you need to go and read the discussion to know why the change is needed. (Or the updated PR description which also mentions it.)

Reply:
Makes sense @srowen @vanzin - updated the patch to shade in `org.sparkproject`, ie no underscore